WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -517,6 +533,15 @@ private extension OrdersViewController {
         let order = resultsController.object(at: indexPath)
         return OrderDetailsViewModel(order: order)
     }
+
+    func getTomorrow() -> Date? {
```

Comment:
This func feels like it should be in a `Date` extension (`Date+Woo.swift`)...like something that could be summoned by calling: 

`Date.tomorrow()`